src/main/java/com/epam/idea/core/security/PasswordHasher.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.idea.core.security;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class PasswordHasher {
+	public static String md5(String str) {
+
+		MessageDigest m = null;
+		try {
+			m = MessageDigest.getInstance("MD5");
+		} catch (NoSuchAlgorithmException e) {
+			e.printStackTrace();
+		}
```

Comment:
It it better to throw unchecked exception because if error occurs then variable 'm' will be null and NullPointerException will be thrown later (line 16).
 try {
			m = MessageDigest.getInstance("MD5");
		} catch (NoSuchAlgorithmException e) {
		        e.printStackTrace();  //use logger instead
			throw new RuntimeException("No Such Algorithm: MD5");
		}